modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSFilters.testcase

```diff
@@ -247,4 +247,25 @@ definition {
 		}
 	}
 
+	@description = "LPS-155304. Assert filter is preloaded when entering on a FDS page for first time."
+	@priority = "5"
+	test WillRenderAsInitialState {
+		task ("Then a filter is preloaded") {
+			VerifyElementPresent(
+				key_filter = "Status: Approved, Draft",
+				locator1 = "FrontendDataSet#FILTER_SUMMARY_LABEL");
+		}
+
+		task ("And the results will be displayed accordingly to the filter selected") {
+			AssertElementPresent(
+				key_itemName = "43466",
+				locator1 = "FrontendDataSet#TABLE_ITEM_ROW");
+		}
```

Comment:
How does this ensure that the results are displayed accordingly? Shouldn't we try and assert some Approved and Draft row items? And that there's no Pending status?

Reply:
Asserting `43466` doesn't give context of what's exactly failing. To me it just seems like a random number. Ideally, if this test fails, reviewer should know what is not displaying correctly and why. ie: filter `approved` failed or filter `draft` failed. If the test fails here, the reviewer still has to figure out what how 43466 relates to which status.

Reply:
Cool. Another thing I thought about to add is verifying the total number of results. In addition to checking Approved/Draft and Pending, another check if we know that the dataset has a fixed number of data is to check that the filtered data is the same number. ie: `20 of 65 results`.

Reply:
> In addition to checking row is Approved/Draft and no Pending is present, another check if we know that the dataset has a fixed number of data is to check that the filtered data is the same number. ie: 20 of 65 results.

Can you also include this validation? It's probably something like `X of 100 results`. After that, looks good to forward for me.